.github/workflows/opam-build.yml

```diff
@@ -46,10 +46,15 @@ jobs:
         run: |
           env HOMEBREW_NO_AUTO_UPDATE=1 brew install tree
 
+      - name: Uninstall zstd?
+        run: sudo apt-get remove --auto-remove zstd
```

Comment:
```suggestion
        run: sudo apt-get remove --auto-remove libzstd-dev
```

Reply:
Actually, the reason this step seems required seems dodgy. I believe a change in merlin is required instead. And in every opam packages that read `.cmi` files for that matter. That’s probably going to push the release a little.

Reply:
I just tried the code suggestion but looks like Dune itself needs lzstd? that seems weird.